modules/monitoring-granting-user-permissions-using-the-cli.adoc

```diff
@@ -6,7 +6,12 @@
 [id="granting-user-permissions-using-the-cli_{context}"]
 = Granting user permissions by using the CLI
 
-You can grant users permissions to monitor their own projects, by using the OpenShift CLI (`oc`).
+You can grant users permissions for the `openshift-monitoring` project or their own projects, by using the OpenShift CLI (`oc`).
```

Comment:
Is something missing here? The [current docs](https://docs.openshift.com/container-platform/4.16/observability/monitoring/enabling-monitoring-for-user-defined-projects.html#granting-user-permissions-using-the-web-console_enabling-monitoring-for-user-defined-projects) say you can grant users permissions **_to monitor_** projects. Should that also appear here?

```suggestion
You can grant users permissions to monitor the `openshift-monitoring` project or their own projects, by using the OpenShift CLI (`oc`).
```